tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/util/BRunUtil.java

```diff
@@ -385,7 +385,7 @@ protected static void invokePackageInit(CompileResult compileResult, String pack
             };
 
             Scheduler scheduler = new Scheduler(4);
-            FutureValue futureValue = scheduler.schedule(jvmArgs, func, null);
+            FutureValue futureValue = scheduler.schedule(jvmArgs, func, null, null, null);
```

Comment:
why do we need this new `schedule()` method if we only pass null for the two additional args?

Reply:
schedule() is being used at Executor.java class (https://github.com/ballerina-platform/ballerina-lang/pull/15561/files#diff-d84a97e503b150e547cc80d097e22155R72). It was improved to accept two additional parameters(callback and property map). This place is one of the methods it is being used and here I pass null as callback and property map is not quite relevant to unit tests.